Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/PersonOverviewViewModel.kt

```diff
@@ -128,13 +156,34 @@ class PersonOverviewViewModel @AssistedInject constructor(
         expirationNotificationService.showNotificationIfStateChanged(ignoreLastCheck = true)
     }
 
+    fun openAdmissionScenarioScreen() = launch {
+        runCatching {
+            // TODO: un-hash admissionCheckScenariosCalculation.getDccAdmissionCheckScenarios()
+            dummy // TODO remove
```

Comment:
I think you should read from the repo here, no ad hoc calculation

Reply:
Please refer to the specs https://github.com/corona-warn-app/cwa-app-tech-spec/blob/3c2b4eb4cb1cb4e9551acc6747c8fea476f98757/docs/spec/dgc-overview-client.md#certificates-tab